### ISSUE TITLE
Withdraw edge case

### ISSUE THREAD

**Issue by anxolin** [gnosis/dex-react#1136]
![image](https://user-images.githubusercontent.com/2352112/85237715-499e6e80-b429-11ea-8fdc-94a42ff96eb3.png)

**Comment by W3stside** [comment]
@anxolin @alfetopito @Velenir @koeppelmann still not clear what a solution here is

**Comment by anxolin** [comment]
This var will contain the last time the user received a token, if it's the current batch, we are in the situation described above.

Ideally we would hide the "Claim" button or/and better handle the exception for the user.

```
mapping(address => mapping(address => uint32)) public lastCreditBatchId;
```

**Comment by anxolin** [comment]
What's the status, is this doable, worth it, not an issue?

**Comment by Velenir** [comment]
Closed by #1302
